lib/scanner/index.js

```diff
@@ -156,7 +157,18 @@ class Scanner {
         this.callback(data);
       })
 
-      .catch(err => console.error(err));
+      .catch(err => {
+        console.info(err);
+        if (this.netErrorListener) {
+          this.netErrorListener(err);
```

Comment:
Should we check if the error is network related? Currently this could be any error.